src/popup/main.js

```diff
@@ -39,21 +39,19 @@ document.addEventListener('DOMContentLoaded', function () {
   submitButton.addEventListener('click', settings.saveSettings);
 
   let ready = false;
-  chrome.runtime.onMessage.addListener(function (
-    request,
-    sender,
-    sendResponse
-  ) {
-    if (request.ready && !ready) {
-      ready = true;
+  chrome.runtime.onMessage.addListener(
+    function (request, sender, sendResponse) {
+      if (request.ready && !ready) {
+        ready = true;
+      }
     }
-  });
+  );
 
-  chrome.storage.local.get(
+  chrome.storage.sync.get(
```

Comment:
Sync defaults to local if chrome doesn't have sync turned on. Otherwise this will sync your data across browsers on different systems.